src/main/io/gps.h

```diff
@@ -132,11 +286,41 @@ typedef struct gpsSolutionData_s {
     uint32_t time;                  // GPS msToW
 } gpsSolutionData_t;
 
+typedef struct ubxVersion_s {
+    uint8_t major;
+    uint8_t minor;
+} ubxVersion_t;
+
+typedef struct ubxSwVersion_s {
+    ubxVersion_t firmwareVersion;
+    ubxVersion_t protocolVersion;
+} ubxSwVersion_t;
+
+/*
+* keeping this table for reference
+typedef enum {
+    UBX_CAP_SAT_NONE = 0x0,
+    UBX_CAP_SAT_GPS = 0x0001,
+    UBX_CAP_SAT_GLO = 0x0002,
+    UBX_CAP_SAT_GAL = 0x0004,
+    UBX_CAP_SAT_BDS = 0x0008,
+    UBX_CAP_SAT_SBAS = 0x0010,
+    UBX_CAP_SAT_QZSS = 0x0020,
+} ubxCapabilities_e;
+*/
+
```

Comment:
This is also unneeded and can be removed

Reply:
It saves us looking up these codes if required in future

Reply:
I get your point, but duplicating the u-blox protocol reference in the code is probably not the goal. Yes, it's annoying to have it now and then "lose" it, but it just makes the code bigger if we never need it.

Once we need it, the dev can do the research in the protocol docs and represent the needed info in the code.

tbh, for someone to want to understand this driver without doing a lot of reading of the protocol docs, is probably not feasible. Maybe we could point people to the docs in a comment up at the top of the file, and have some expectation that a dev working more deeply on the workings of the driver would do some homework?